src/main/java/competition/subsystems/hood/commands/HoodMaintainerCommand.java

```diff
@@ -33,9 +37,38 @@ public void initialize(){
     }
 
     @Override
-    protected void calibratedMachineControlAction(){
+    protected void calibratedMachineControlAction() {
+        if (!this.hood.isCalibrated()) {
```

Comment:
This block won't be evaluated - `calibratedMachineControlAction` is only ever called if the system is in a calibrated state.

Reply:
Will leave for safety ⚠️